module/data/item/templates/equippable-item.mjs

```diff
@@ -79,7 +79,7 @@ export default class EquippableItemTemplate extends SystemDataModel {
   get equippableItemChatProperties() {
     foundry.utils.logCompatibilityWarning("EquippableItemTemplate#equippableItemChatProperties is deprecated. "
       + "Please use EquippableItemTemplate#equippableItemCardProperties.",
-      { since: "DnD5e 3.0", until: "DnD5e 3.2", once: true });
+    { since: "DnD5e 3.0", until: "DnD5e 3.2", once: true });
```

Comment:
```javascript
    foundry.utils.logCompatibilityWarning(
      "EquippableItemTemplate#equippableItemChatProperties is deprecated. "
      + "Please use EquippableItemTemplate#equippableItemCardProperties.",
      { since: "DnD5e 3.0", until: "DnD5e 3.2", once: true });
```